ring_buffer/ring_buffer.py

```diff
@@ -1,11 +1,19 @@
 class RingBuffer:
-  def __init__(self, capacity):
-    self.capacity = capacity
-    self.current = 0
-    self.storage = [None]*capacity
+    def __init__(self, capacity):
+        self.capacity = capacity
+        self.current = 0
+        self.storage = [None]*capacity
 
-  def append(self, item):
-    pass
+    def append(self, item):
+      # checks to see if the current is less than the ring buffer capacity, if it is then it appends it to the end of the list
+        if self.current < self.capacity:
+            self.storage[self.current] = item
+            self.current += 1
+      # else, it overwrites the oldest value in the list, starting with index 0, then increments it to be ready to overwrite the next value in the list
+        else:
+            self.current = 0
```

Comment:
Close. We want to overwrite the last one as well so if it is at it's capacity we want to overwrite that one and THEN we want to set current to 0